suzieq/sqobjects/basicobj.py

```diff
@@ -72,6 +72,7 @@ def __init__(self, engine_name: str = 'pandas',
             self.view = view or 'latest'
 
         self.columns = columns
+        self.default_columns = ['hostname']
```

Comment:
is default_columns the right name? That'd gibe another contributor the impression that this is used for all commands. Is _unique_def_column a better name? We also start var names that we use for internal use only with _, and so is _unique_def_column a better name?

Reply:
You're right, it could be confusing to other contributors. It's better to go with `_unique_def_column`.